ClinicWeb/src/main/java/pl/infoshare/clinicweb/doctor/DoctorController.java

```diff
@@ -114,52 +129,71 @@ public String searchDoctorByPesel(@ModelAttribute Doctor doctor) {
 
     @PostMapping("/search-doctor")
     public String searchDoctorByPesel(@RequestParam(value = "id", required = false) long id, Model model) {
+        log.info("Invoked searchDoctorByPesel method with id: {}", id);
 
         DoctorDto doctorById = doctorService.findById(id);
-
+        if (doctorById != null) {
```

Comment:
dobrą praktyką jest logować tylko kiedy nie znajdziemy jakiegoś rekordu, ponad to można tę informację przenieść do serwisu do metody findById